hedera-node/src/test/java/com/hedera/services/store/models/AccountTest.java

```diff
@@ -92,6 +93,46 @@ void canAssociateWithNewToken() {
 		assertEquals(expectedFinalTokens, assocTokens.toReadableIdList());
 	}
 
+	@Test
+	void accountEqualsCheck() {
+		// setup:
+		var account = new Account(subjectId);
+		account.setAssociatedTokens(assocTokens);
+		account.setExpiry(1000L);
+		account.initBalance(100L);
+		account.setOwnedNfts(1L);
+		account.incrementOwnedNfts();
+
+		subject.setExpiry(1000L);
+		subject.initBalance(100L);
+		subject.setOwnedNfts(1L);
+		subject.incrementOwnedNfts();
+
+		// when:
+		var actualResult = subject.equals(account);
+
+		// expect:
+		assertEquals(account.getOwnedNfts(), subject.getOwnedNfts());
+		// and:
+		assertEquals(account.getId(), subject.getId());
+		// and:
+		assertEquals(account.getAssociatedTokens(), subject.getAssociatedTokens());
+		// and:
+		assertTrue(actualResult);
+	}
+
+	@Test
+	void accountHashCodeCheck() {
+		// setup:
+		subject.incrementOwnedNfts();
+
+		// when:
+		var actualResult = subject.hashCode();
+
+		// expect:
+		assertEquals(-1217798680, actualResult);
```

Comment:
Normally when you test hashCode you create another instance of the subject class and verify that if the same properties are populated the hashcode is equal. The test you wrote will brake if you add another property. 

My assumption is that you executed `hashCode` and saw what the output is and then placed the number here, which is basically testing the code with the output from the code and that's not meaningful